monocle/web_utils.py

```diff
@@ -138,10 +138,12 @@ def get_spawnpoint_markers():
         spawns = session.query(Spawnpoint)
         return [{
             'spawn_id': spawn.spawn_id,
-            'despawn_time': spawn.despawn_time,
+            'despawn_time': spawn.despawn_time
+                if spawn.updated and spawn.updated > conf.LAST_MIGRATION else None,
```

Comment:
I'd say truth-testing `spawn.updated` before the comparison isn't necessary. The default migration time isn't zero and if someone wants to disable migration checks entirely they should set `LAST_MIGRATION` to `1` or any time before their first scan. So with a valid configuration `spawn.updated > conf.LAST_MIGRATION` should always fail when `spawn.updated` fails the truth test.